installation/install

```diff
@@ -85,11 +81,12 @@ function handle_error
 
 function get_os_type
 {
-	which zypper &> /dev/null && os="opensuse"
 	which pacman &> /dev/null && os="archlinux"
-	which dnf &> /dev/null && os="fedora"
 	which apt-get &> /dev/null && os="debian"
+	which dnf &> /dev/null && os="fedora"
 	which emerge &> /dev/null && os="gentoo"
+	which zypper &> /dev/null && os="opensuse"
```

Comment:
Why did you move opensuse & fedora position ?

Basically, when you make a pull request, you should modify or add lines related with the issue you are resolving

In this case, you should not move/change lines that are not related to `add support for solus`.